pkg/adapter/adapter.go

```diff
@@ -152,20 +153,29 @@ func (a *mongoDbAdapter) makeCloudEvent(data bson.M) (*cloudevents.Event, error)
 	// Create Event.
 	event := cloudevents.NewEvent(cloudevents.VersionV1)
 
-	// Set cloud event specs and attributes. TODO: issue #43
+	// Set cloud event specs and attributes.
 	// 		ID     -> id of mongo change object
 	// 		Source -> Source environment variable.
-	// 		Type   -> type of change either insert, delete or update.
+	// 		Type   -> type of change either inserted, deleted or updated.
 	//		Data   -> data payload containing either id only for
 	//                deletion or full object for other changes.
 	change, err := utils.DecodeChangeBson(data)
 	if err != nil {
 		return nil, fmt.Errorf("error decoding bson change object: %w", err)
 	}
 	event.SetID(change.ID)
-	event.SetType(change.OperationType)
 	event.SetSource(a.ceSource)
 	event.SetData(cloudevents.ApplicationJSON, change.Payload)
+	switch change.OperationType {
```

Comment:
you can have a map of mongo types to our types and the code would be cleaner... 
but this is optional, not need to do it if you thing it adds complexity

Reply:
done